simapp/cmd/simcli/main.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/cosmos/cosmos-sdk/std"
```

Comment:
nit: grouping is incorrect.

Reply:
Does `make format` do this automatically?